petl/test/helpers.py

```diff
@@ -8,11 +8,14 @@
 def ieq(expect, actual, cast=None):
     ie = iter(expect)
     ia = iter(actual)
-    for e, a in izip_longest(ie, ia, fillvalue=None):
-        if cast:
-            a = cast(a)
-        eq_(e, a)
-        
+    try:
+        for e, a in izip_longest(ie, ia, fillvalue=None):
+            if cast:
+                a = cast(a)
+            eq_(e, a)
+    except:
+        return
```

Comment:
If this is needed because there is code inside petl that raises StopIteration from inside a generator, then I think it would be better to fix the code rather the test. To deal with PEP 479 change I believe we can just replace "raise StopIteration" with a bare "return" instead.